praetorian_cli/sdk/entities/account_discovery.py

```
@@ -0,0 +1,397 @@
+                f'{base_url}/my',
+                headers=headers,
+                params={'key': '#capability_schedule#'},
+                timeout=30,
+            )
+            if resp.status_code != 200:
+                logger.debug('Schedule load for %s returned status %d', email, resp.status_code)
+                return []
+            data = resp.json()
+            schedules = data.get('capabilityschedules', [])
+            return [(sched, acct) for sched in schedules]
+        except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)